rundeckapp/grails-spa/packages/ui-trellis/src/library/components/widgets/rundeck-info/RundeckInfo.spec.ts

```diff
@@ -0,0 +1,97 @@
+  url: jest.fn().mockReturnValue("http://localhost"),
+}));
+const mountRundeckInfo = async (props = {}) => {
+  return mount(RundeckInfo, {
+    props: {
+      appInfo: { title: "Rundeck", logocss: "some-css" },
+      version: {
+        number: "1.0.0",
+        tag: "stable",
+        name: "Erebus",
+        color: "red",
+        icon: "glass",
+        edition: "Community",
+      },
+      server: {
+        name: "localHost",
+        icon: "paperclip",
+        uuid: "uuid1",
+      },
+      latest: {
+        title: "Rundeck",
+        full: "v5.2.0-20240410",
+        number: "5.2.0",
+        tag: "GA",
+        color: "aquamarine",
+        date: new Date(),
+        icon: "knight",
+        edition: "Community",
+      },
+      ...props,
+    },
+  });
+};
+describe("RundeckInfo", () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+  it("renders RundeckLogo when appInfo.title is 'Rundeck'", async () => {
+    const wrapper = await mountRundeckInfo();
+    expect(wrapper.findComponent({ name: "RundeckLogo" }).exists()).toBe(true);
+    expect(wrapper.findComponent({ name: "PagerdutyLogo" }).exists()).toBe(
+      false,
+    );
+  });
+  it("renders PagerdutyLogo when appInfo.title is 'Pagerduty'", async () => {
+    const wrapper = await mountRundeckInfo({ appInfo: { title: "Pagerduty" } });
+    expect(wrapper.findComponent({ name: "PagerdutyLogo" }).exists()).toBe(
+      true,
+    );
+    expect(wrapper.findComponent({ name: "RundeckLogo" }).exists()).toBe(false);
+  });
+  it("renders RundeckVersions information with correct props", async () => {
+    const wrapper = await mountRundeckInfo();
+
+    const rundeckVersionComponent = wrapper.findComponent({
+      name: "RundeckVersion",
+    });
+    expect(rundeckVersionComponent.exists()).toBe(true);
+
+    expect(rundeckVersionComponent.find("span").text()).toBe("Rundeck 1.0.0");
+  });
+  it("renders ServerDisplay correctly with default props", async () => {
+    const wrapper = await mountRundeckInfo();
+    const serverComponent = wrapper.findComponent({ name: "ServerDisplay" });
+    expect(serverComponent.exists()).toBe(true);
+
+    expect(
+      serverComponent.find('[data-test-id="server-title"]').attributes("title"),
+    ).toBe("paperclip-uu / uuid1");
+  });
+
+  it("renders version display correctly", async () => {
+    const wrapper = await mountRundeckInfo();
+    const versionDisplayComponent = wrapper.findComponent({
+      name: "VersionDisplay",
+    });
+    expect(versionDisplayComponent.props("text")).toBe("Erebus red glass");
+  });
+  it("renders the latest release information correctly", async () => {
+    const wrapper = await mountRundeckInfo();
+    const rundeckVersionComponent = wrapper.findComponent({
+      name: "RundeckVersion",
+    });
+    expect(rundeckVersionComponent.exists()).toBe(true);
+    expect(rundeckVersionComponent.find("span").text()).toBe("Rundeck 1.0.0");
```

Comment:
RundeckInfo renders 2 RundeckVersion components - one that you've tested before, that will render "Rundeck 1.0.0", and one that renders the latest release info. 

This test should be checking the text of the second RundeckVersion. To do so, you can use the method [findAllComponents](https://test-utils.vuejs.org/api/#findAllComponents) as findComponent will always return the first occurrence that matches what you're searching.